src/BuildIntegration/findvcvarsall.bat

```diff
@@ -0,0 +1,23 @@
+@ECHO OFF
+SETLOCAL
+
+SET vswherePath=%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+IF NOT EXIST "%vswherePath%" GOTO :ERROR
+
+FOR /F "tokens=*" %%i IN (	'
+	  "%vswherePath%" -latest -products * ^
```

Comment:
Please add the ```-prerelease``` flag, which includes searching for prerelease builds of Visual Studio.